src/elements/content-sidebar/Sidebar.js

```diff
@@ -46,17 +47,22 @@ type Props = {
 };
 
 type State = {
-    isDirty: boolean,
-    isOpen: boolean,
+    isOpen: boolean, // Local isOpen state consists of stored forced state (if any) and responsive adjustments
 };
 
+export const SIDEBAR_FORCE_KEY: 'bcs.force' = 'bcs.force';
```

Comment:
It looks like our abstraction is using `/` under the hood as a delimiter, should we continue that here?

`${KEY_PREFIX}/${SERVICE_VERSION}/${key}`

Reply:
I actually think dot notation has the advantage here because it could be used with lodash/get.